src/service/action-impl.js

```diff
@@ -342,16 +342,16 @@ export class ActionService {
   /**
    * @param {!Element} target
    * @param {string} actionEventType
-   * @return {?{node: !Element, actionInfo: !ActionInfoDef}}
+   * @return {?{node: !Element, actionsInfo: Array<!ActionInfoDef>}}
    */
-  findAction_(target, actionEventType) {
+  findActions_(target, actionEventType) {
```

Comment:
Nit: Keep the original name. If it doesn't return an array, it should probably be singular.